packages/iTwinUI-react/src/core/Wizard/Stepper.tsx

```diff
@@ -0,0 +1,94 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils';
+import '@itwin/itwinui-css/css/stepper.css';
+import { StepProperties } from './Wizard';
```

Comment:
I am not 100% sure but this might cause circular dependencies import.

Reply:
`Stepper` no longer imports anything from `Wizard` so this should be good